python/nammu/SOAPClient/SOAPClient.py

```diff
@@ -0,0 +1,62 @@
+import requests
+import logging
+import httplib as http_client
+from HTTPRequest import HTTPRequest
+
+# Dunno if this is the right place to set up logging
+logging.basicConfig()
```

Comment:
Yeah, that's probably the place. I just got a little confuse by some examples that were doing it like this. I'll move it.